packages/redux/src/raffles/selectors.ts

```diff
@@ -0,0 +1,89 @@
+import { getEntities, getEntityById } from '../entities/selectors';
+import {
+  getError as getErrorFromReducer,
+  getIsLoading as getIsLoadingFromReducer,
+} from './reducer';
+import type {
+  Raffle,
+  RaffleParticipation,
+} from '@farfetch/blackout-client/src';
+import type { RafflesState } from './types';
+import type { StoreState } from '../types/storeState.types';
+
+/**
+ * Returns all raffles from state.
+ *
+ * @param state - Application state.
+ *
+ * @returns Object with key values pairs representing raffleId and raffle properties.
```

Comment:
This description could be improved though.

Reply:
done